wdl/DepthPreprocessing.wdl

```diff
@@ -141,7 +141,7 @@ task MergeSample {
   input {
     File gcnv
     Array[File] cnmops
-    Float max_dist = 0.25
+    Float? max_dist = 0.25
```

Comment:
Change to just `Float? max_dist`? Assigning the value is redundant with the line below.

Reply:
Yep, actioned so resolving.